api/boilerplate/views.py

```diff
@@ -17,3 +27,50 @@ class GroupViewSet(viewsets.ModelViewSet):
     """
     queryset = Group.objects.all()
     serializer_class = GroupSerializer
+    
+def open_id_auth(request):
+
+    redirect_to='https://eid.local/api/eIdService/init'
+    params = None
+    if request.method == "GET":
+        params = request.GET
+    elif request.method == "POST":
+        params = request.POST
+    
+    return HttpResponseRedirect(redirect_to + '?' + params.urlencode())
```

Comment:
I wouldn't do it that way. You are hardcoding the hostname. Change
```
redirect_to = '/api/eIdService/init'
```
This works and you don't have to specify the hostname.

Reply:
Thank you very much for the review! Done and tested, works fine!